src/stars.css

```diff
@@ -0,0 +1,30 @@
+.stars {
+  display: flex;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars__text {
```

Comment:
idk is it a mistake, but for my opinion stars is independent block. if you replace review text inside you made it stars part. I recomended to create another block and replace stars and review inside it. but it`s only my opinion so better ask mentors